app/chat_endpoint_handlers.py

```
@@ -4,6 +4,7 @@
 from bson import json_util
 from fastapi import BackgroundTasks
 
+from main import remaining_messages
```

Ruff (0.12.2)

`main.remaining_messages` imported but unused

Remove unused import: `main.remaining_messages`

(F401)